### PR TITLE
Feature/#611 flymble mobile

### PR DESCRIPTION
**Pull request by szczepanbarszczowski**

### CODE REVIEW

File: src/routes/Booksley/Booksley.js [hunk 1]
```diff
@@ -19,222 +20,230 @@ import secondPreview from './assets/secondCover.png'
...
       </div>
+    </div>
+
+    <div className={c.sectionWithBorderWrapper}>
+      <div className={c.tenthSection}>
+        <div className={c.textWrapper}>
+          <div className={cn(c.title, c.booksTitle)}>
+            Judge a book<br />by its cover
+          </div>
+          <div className={cn(c.subtitle)}>Adaptive screens</div>
```
Comment (CodeMeNatalie) [inline src/routes/Booksley/Booksley.js]: no need of `classes`

File: src/routes/Booksley/Booksley.module.scss [hunk 2]
```diff
@@ -6,16 +6,33 @@
...
   .col {
     @include flex(column, flex-start, center);
 
     .booksTitle {
       padding-bottom: 21px;
       line-height: 58px;
+
+      @include media-breakpoint-down(md) {
+        line-height: 24px;
+        @include pb(1);
```
Comment (CodeMeNatalie) [inline src/routes/Booksley/Booksley.module.scss]: mixins before declarations

File: src/routes/Booksley/Booksley.module.scss [hunk 3]
```diff
@@ -51,6 +84,11 @@
 
   padding-top: 95px;
   width: 405px;
+
+  @include media-breakpoint-down(md) {
+    @include p(2);
+    @include pb(3);
```
Comment (CodeMeNatalie) [inline src/routes/Booksley/Booksley.module.scss]: `@include p(2, 2, 3);` or `@include p(2, 2, 3, 2);`

File: src/routes/Booksley/Booksley.module.scss [hunk 4]
```diff
@@ -286,34 +456,82 @@
...
+    @include flex(column, center, space-between);
+    @include pt(2);
+  }
+
   .imageWrap {
-    width: 440px;
+    @include px(1);
+    width: 230px;
+
+    &:first-child {
```
Comment (CodeMeNatalie) [inline src/routes/Booksley/Booksley.module.scss]: If there are only 3 images =>
`& * { width: 100%; }`

It should take each child. Please try it.

File: src/routes/Flymble/Flymble.module.scss [hunk 5]
```diff
@@ -107,22 +131,41 @@
...
-  line-height: 30px;
+  font-size: 20px;
+  line-height: 34px;
 }
 
 .col {
   @include col(6, $gutter: false);
+
+  @include media-breakpoint-down(md) {
+    @include col(12, $gutter: false);
```
Comment (CodeMeNatalie) [inline src/routes/Flymble/Flymble.module.scss]: `@include col($gutter: flase);`

File: src/routes/HowAboutSales/HowAboutSales.module.scss [hunk 6]
```diff
@@ -23,24 +23,44 @@
...
+
+    @include media-breakpoint-down(md) {
+      @include pt(2);
+    }
   }
 
   .header,
   .headerWithBorder,
   .itFitsHeader {
+    @include size(100%, 100%);
```
Comment (CodeMeNatalie) [inline src/routes/HowAboutSales/HowAboutSales.module.scss]: `@include size(100%);`

File: src/components/CaseStudiesElements/HeroSection/HeroSection.module.scss [hunk 7]
```diff
@@ -4,14 +4,26 @@
...
     font-family: $font-family-medium;
+
+    @include media-breakpoint-down(md) {
+      @include font(h4, $color-black, $font-family: $font-family-medium);
+    }
   }
 
   .subtitle {
     @include font(h1-light, $gray-header);
     padding-bottom: 50px;
```
[the remark this answers is not in the document]
Comment (szczepanbarszczowski) [reply src/components/CaseStudiesElements/HeroSection/HeroSection.module.scss]: +1

File: src/routes/Booksley/Booksley.js [hunk 8]
```diff
@@ -19,222 +20,230 @@ import secondPreview from './assets/secondCover.png'
...
-            Let’s make a book<br />exchange club
-          </div>
-          <div className={cn(c.subtitle, c.booksSubtitle)}>The brief</div>
-          <span className={c.text}>
-            Booksley is about swapping books. No cash, no<br />
-            payments, just pure trade. You want a book<br />
-            someone has? Offer your books in exchange!
-          </span>
+    <div className={c.secondSection}>
+      <div className={cn(c.col)}>
```
[the remark this answers is not in the document]
Comment (szczepanbarszczowski) [reply src/routes/Booksley/Booksley.js]: +1

File: src/routes/Booksley/Booksley.js (hunk 1 above)
Comment (szczepanbarszczowski) [reply src/routes/Booksley/Booksley.js]: +1

File: src/routes/Booksley/Booksley.module.scss (hunk 2 above)
Comment (szczepanbarszczowski) [reply src/routes/Booksley/Booksley.module.scss]: +1

File: src/routes/Booksley/Booksley.module.scss (hunk 3 above)
Comment (szczepanbarszczowski) [reply src/routes/Booksley/Booksley.module.scss]: +1

File: src/routes/Booksley/Booksley.module.scss (hunk 4 above)
Comment (szczepanbarszczowski) [reply src/routes/Booksley/Booksley.module.scss]: +1

File: src/routes/Flymble/Flymble.module.scss (hunk 5 above)
Comment (szczepanbarszczowski) [reply src/routes/Flymble/Flymble.module.scss]: +1

**Pull request merged**